RxCocoa/iOS/UISearchController+Rx.swift

```diff
@@ -65,6 +78,11 @@
                 .map {_ in}
         }
         
+        /// Reactive wrapper for `searchResultsUpdater` message.
+        public var updateSearchResultsFor: Observable<UISearchController> {
```

Comment:
I think this should be `ControlEvent` probably. I don't think we need `for`, just `updateSearchResults`.